packages/console/src/components/CreateConnectorForm/Footer/index.tsx

```diff
@@ -31,35 +33,51 @@ function Footer({
   onClickCreateButton,
 }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console.upsell.paywall' });
-  const { currentPlan } = useContext(SubscriptionDataContext);
+  const { currentPlan, currentSku, currentSubscriptionUsage, currentSubscriptionQuota } =
+    useContext(SubscriptionDataContext);
 
   const standardConnectorCount = useMemo(
     () =>
-      existingConnectors.filter(
-        ({ isStandard, isDemo, type }) => isStandard && !isDemo && type === ConnectorType.Social
-      ).length,
+      isDevFeaturesEnabled
+        ? // No more standard connector limit in new pricing model.
+          0
+        : existingConnectors.filter(
+            ({ isStandard, isDemo, type }) => isStandard && !isDemo && type === ConnectorType.Social
+          ).length,
     [existingConnectors]
   );
 
   const socialConnectorCount = useMemo(
     () =>
-      existingConnectors.filter(
-        ({ isStandard, isDemo, type }) => !isStandard && !isDemo && type === ConnectorType.Social
-      ).length,
-    [existingConnectors]
+      isDevFeaturesEnabled
+        ? currentSubscriptionUsage.socialConnectorsLimit
+        : existingConnectors.filter(
+            ({ isStandard, isDemo, type }) =>
+              !isStandard && !isDemo && type === ConnectorType.Social
+          ).length,
+    [existingConnectors, currentSubscriptionUsage.socialConnectorsLimit]
   );
 
-  const isStandardConnectorsReachLimit = hasReachedQuotaLimit({
-    quotaKey: 'standardConnectorsLimit',
-    plan: currentPlan,
-    usage: standardConnectorCount,
-  });
+  const isStandardConnectorsReachLimit = isDevFeaturesEnabled
+    ? // No more standard connector limit in new pricing model.
+      false
+    : hasReachedQuotaLimit({
+        quotaKey: 'standardConnectorsLimit',
+        plan: currentPlan,
+        usage: standardConnectorCount,
+      });
 
-  const isSocialConnectorsReachLimit = hasReachedQuotaLimit({
-    quotaKey: 'socialConnectorsLimit',
-    plan: currentPlan,
-    usage: socialConnectorCount,
-  });
+  const isSocialConnectorsReachLimit = isDevFeaturesEnabled
+    ? hasReachedSubscriptionQuotaLimit({
+        quotaKey: 'socialConnectorsLimit',
+        usage: currentSubscriptionUsage.socialConnectorsLimit,
+        quota: currentSubscriptionQuota,
```

Comment:
I think maybe we don't have to pass `currentSubscriptionQuota` into the function, as this can be fetched from the context.

Reply:
At this stage, `hasReachedSubscriptionQuotaLimit()` is a function with out any context, and relies purely on the input. We can refactor the code and attach context (e.g. `subscriptionQuota` to the method) in the future update. See LOG-9696.

Reply:
ok